grader/src/test/java/edu/pdx/cs410J/grader/scoring/ProjectSubmissionOutputFileParserTest.java

```diff
@@ -185,7 +183,10 @@ public void parseCompilerTestWithNoOutput() {
     file.line();
     file.line("*****  Test 1: No arguments");
 
-    ProjectSubmission submission = parse(file);
+    ProjectSubmission submission = null;
```

Comment:
Do you think that splitting the declaration and assignment make the code easier to read?  I'm not so sure.